### PR TITLE
Remove TTLSecondsAfterFinished from Eventing jobs

### PR DESCRIPTION
**Pull request by pierDipi**
In order to have privileged service accounts always bound to jobs and to avoid any compliance system flagging unbound SAs, we will keep completed jobs (one per version).

In the future, we might clean up previous jobs automatically

### CODE REVIEW

File: openshift-knative-operator/pkg/common/job.go [hunk 1]
```diff
@@ -31,3 +31,19 @@ func VersionedJobNameTransform() mf.Transformer {
 		return nil
 	}
 }
+
+func RemoveTTLSecondsAfterFinished() mf.Transformer {
```
[the remark this answers is not in the document]
Comment (pierDipi) [reply openshift-knative-operator/pkg/common/job.go:35]: it's WIP

[the remark this answers is not in the document]
Comment (pierDipi) [reply openshift-knative-operator/pkg/common/job.go]: Now it's ready and tested

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**